sources/documents/2_RTB/PianoDiQualifica/tabelle_M3.tex

```diff
@@ -20,8 +20,9 @@
 		\hline
 		\endlastfoot
         
-    M.3.1 & Rischi non previsti in assoluto per sprint & < 5 & < 2 \\
+    M.3.1 & Rischi inattesi & < 2 & < 0 \\
```

Comment:
Qua te lo segno, ma è abbastanza opzionale: penso si potrebbero mettere delle unità di misura ad ogni metrica. Tipo in questo caso aggiungere che si tratta di rischi

Reply:
Non saprei che unità di misura indicare, il numero è ottenuto da un rapporto tra il numero di rischi non previsti e il numero di rischi massimi attesi.
Cosa intendi con:
> Tipo in questo caso aggiungere che si tratta di rischi